app_scheduler/AppScheduleEntry[] entries
